OnPod/FlightComputer/SetupScriptPlusDockerfile/README.md

```diff
@@ -0,0 +1,30 @@
+# README.md
+install docker // differes per platform, google it
+
+cd into folder with Dockerfile
+
+" docker build -t pi_fc:v1 . " 
```

Comment:
Super trivial change. Might be better for readability to wrap command using (\`) instead of quotes.
IE: `docker build -t pi_fc:v1 .`